src/utils/build-schema-with-resolvers.ts

```diff
@@ -0,0 +1,70 @@
+import { GraphQLObjectType, GraphQLScalarType, GraphQLInterfaceType, GraphQLUnionType, GraphQLEnumType, buildASTSchema, DocumentNode, parse } from 'graphql';
+import { mergeTypeDefs, mergeResolvers } from '../epoxy';
+import { ResolversComposerMapping, composeResolvers } from './resolvers-composition';
+
+export interface BuildSchemaWithResolversOptions {
+    typeDefs: string | string[] | DocumentNode | DocumentNode[];
+    resolvers?: any;
+    resolversComposition?: ResolversComposerMapping;
+}
+
+export function buildSchemaWithResolvers({ typeDefs, resolvers, resolversComposition }: BuildSchemaWithResolversOptions) {
+    if (typeDefs instanceof Array) {
+        typeDefs = mergeTypeDefs(typeDefs);
```

Comment:
I'm not sure that this method should do schema merging. maybe it should get the ready-to-use types?